extensions/wikia/PortableInfobox/PortableInfobox.i18n.php

```diff
@@ -35,6 +40,12 @@
 	'portable-infobox-xml-parse-error-tag-name-mismatch' => 'XML Error: Opening and ending tag mismatch (for example: <data></label>)',
 	'portable-infobox-xml-parse-error-tag-not-finished' => 'XML Error: premature end of tag',
 	'portable-infobox-xml-parse-error-infobox-tag-attribute-unsupported' => 'Unsupported attribute used inside <infobox> tag. $1 param contains attribute name.',
+	'portable-infobox-builder-entry-point-modal-title' => 'Title for portable infobox builder entry point modal on
+	edit page with a question: Do you want to create infobox?',
+	'portable-infobox-builder-entry-point-modal-message' => 'Message in portable infobox builder asking if user wants
+	 to create an infobox and if she / he would like to use new Infobox Builder UI tool.',
+	'portable-infobox-builder-entry-point-modal-ok-button' => 'Ok button',
```

Comment:
I was thinking about whether it's worth to make it even more explicit ("Ok button, meaning the user confirms using the new infobox builder")?